### ISSUE TITLE
Bookmarklet asks for location either not enough, or too much

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/backend#386]
_bus pun_
- on Firefox, it seems to ask twice for locations if you're supporting / creating requests
- on Chrome, it seems to not ask that much (or at all)

@meganwaps can you try and explore this problem. E.g adding in Safari & giving more depth to my descriptions. More data is needed before Mark can take this on.

**Comment by JosephMcArthur** [comment]
If this is actually an issue (pending @meganwaps testing), my preferred route to closing would be removing the ask for location completely. @markmacgillivray is it possible to get location data (accurate) to the country level passively upon sign up?

**Comment by JosephMcArthur** [comment]
@markmacgillivray assigning to you. I think we should just take out the location ask here. But would like to know alternative places we could get some location info (e.g sign up).

**Comment by JosephMcArthur** [comment]
This doesn't need to be in /requests/new

**Comment by markmacgillivray** [comment]
Would asking for location on sign up seem a bit weird? It is not really clear why location would be needed at sign up - I would wonder why oabutton needs my location during a sign up process. Asking whilst using the bookmarklet / using the requests page may also not be totally clear, but at least it relates to the act of creating a request.

I can easily remove the location requesting from the plugin and the new request page, but I don't think we have an agreed alternative for collecting location data another way, and perhaps not even an agreed idea of why we are collecting location data - it is not actually necessary for the use case of the user getting access to something.

So I will leave this as is until discussion continues, but just let me know if / when you want the location requests removed from bookmarklet and new request, even if before we have an alternative.

**Comment by markmacgillivray** [comment]
This is straightforward to do once we make the decision - assigning to @JosephMcArthur to make him think about the above :)

**Comment by JosephMcArthur** [comment]
- [ ] Remove it from the bookmarklet & /request/new page. We'll keep it in the plugins. 

- [ ] Once that is done, we'll close this & start up a separate issue for discussion of how to get it instead.

**Comment by markmacgillivray** [comment]
OK, it has been removed and pushed to live.

**Comment by JosephMcArthur** [comment]
@chealsye can we add a note on this to our mini post on making it simpler
to use ( easier nav etc)

On Fri, 23 Dec 2016, 1:26 p.m. markmacgillivray, <notifications@github.com>
wrote:

Assigned #386 <https://github.com/OAButton/backend/issues/386> to
@JosephMcArthur <https://github.com/JosephMcArthur>.

—
You are receiving this because you were mentioned.


Reply to this email directly, view it on GitHub
<https://github.com/OAButton/backend/issues/386#event-904059423>, or mute
the thread

**Comment by JosephMcArthur** [comment]
Thank you Mark. We'll roll discussions on where to get location data into the sign up issue #432. Seems like that is the next best place to discuss.
